crypto/chacha/build.info

```diff
@@ -54,3 +54,4 @@ GENERATE[chacha-ia64.S]=asm/chacha-ia64.pl
 GENERATE[chacha-ia64.s]=chacha-ia64.S
 GENERATE[chacha-loongarch64.S]=asm/chacha-loongarch64.pl
 GENERATE[chacha-riscv64-zvkb.s]=asm/chacha-riscv64-zvkb.pl
+GENERATE[chacha-riscv64-v.s]=asm/chacha-riscv64-v.pl
```

Comment:
I would like to use a single perl generator with the additional argument for `vror` instruction without the new additional perl file.

e.g.
```
GENERATE[chacha-riscv64-zvkb.s]=asm/chacha-riscv64-rvv.pl --enable-zvkb=true
GENERATE[chacha-riscv64-rvv.s]=asm/chacha-riscv64-rvv.pl --enable-zvkb=false
```

Reply:
Thanks. @ZenithalHourlyRate also commented on this.